docs: add matplotlib histogram to cookbook (#143)
best-practice-and-impact/afcharts-py: src/afcharts/cookbook/01-matplotlib-usage.qmd

> This histogram uses the afcharts theme, and shows the distribution of life expectancy by number of countries. There are pale grey grid lines extending out from the y axis. The bars are dark blue with white space between each.

Reviewer: I know this isn't yours but I just noticed this alt text is completely wrong! Could you please update it to match the R cookbook, i.e.

```suggestion
This stacked bar chart uses the afcharts theme and shows the proportions of countries with a life expectancy over and under 75 by continent. The continents are listed along the x axis, with the y axis labelled between 0% and 100%. The colours for the bar segments are from the main Analysis Function palette - dark blue for under 75 and orange for over 75, denoted by a legend at the bottom of the chart. There is whitespace between each bar.
```

</div>

> ## Historgrams

Reviewer: Typo

```suggestion
## Histograms
```

```{python}

#| output: false
import matplotlib.pyplot as plt
from plotly.express.data import gapminder

# Set default theme
plt.style.use("afcharts.afcharts")

# Filter for year 2007
df = gapminder().query("year == 2007")

fig = plt.figure()

plt.hist(
    df["lifeExp"],
    bins=range(int(df["lifeExp"].min()), int(df["lifeExp"].max()) + 5, 5),
    edgecolor="white"
)

> plt.xlabel(None)

Reviewer: This line isn't necessary: matplotlib won't add an axis title unless you specify one

> plt.ylabel("Number of countries")

Reviewer: All text should be horizontal to meet AF guidelines

```suggestion
plt.ylabel("Number of\n countries", rotation=0, labelpad=40)
```
